rust/src/api.rs

```diff
@@ -69,7 +66,14 @@ pub fn get_address() -> Result<Address> {
 pub fn open_channel(peer_pubkey_and_ip_addr: String, channel_amount_sat: u64) -> Result<()> {
     let peer_info = parse_peer_info(peer_pubkey_and_ip_addr)?;
     let rt = Runtime::new()?;
-    rt.block_on(async { wallet::open_channel(peer_info, channel_amount_sat).await })
+    rt.block_on(async {
+        let _ = wallet::open_channel(peer_info, channel_amount_sat).await;
+        loop {
+            // looping here indefinitely to keep the connection with the maker alive.
+            std::thread::sleep(Duration::new(u64::MAX, u32::MAX));
```

Comment:
This will actually break, I've fixed it on my branch

Reply:
why?